packages/core/test/tree/treeTests.tsx

```diff
@@ -109,6 +110,14 @@ describe("<Tree>", () => {
         TestUtils.Simulate.click(document.query(`.c3 > .${Classes.TREE_NODE_CONTENT} .${Classes.TREE_NODE_CARET}`));
         assert.isTrue(onNodeCollapse.calledOnce);
         assert.deepEqual(onNodeCollapse.args[0][1], [3]);
+
+        assert.isTrue(onNodeContextMenu.notCalled);
+        // TestUtils.Simulate.contextMenu is a function, just not included in the typings
+        // nonetheless, the below line causes React to throw an error for some reason
+        // (TestUtils.Simulate as any).contextMenu(document.query(`.c0 > .${Classes.TREE_NODE_CONTENT}`));
+
+        // assert.isTrue(onNodeContextMenu.calledOnce);
+        // assert.deepEqual(onNodeContextMenu.args[0][1], [0]);
```

Comment:
Ha, I actually forgot that I wrote `Tree`! Follows the old programmer's aphorism that you should write your own code well, because you'll forget it quickly and it'll look just as foreign as anyone else's code.